src/js/providers/youtube.js

```diff
@@ -247,6 +247,10 @@ define([
             switch (_youtubeState) {
 
                 case youtubeStates.UNSTARTED: // -1: //unstarted
+                    // play video on android to avoid being stuck in this state
+                    if (utils.isAndroid()) {
+                        _youtubePlayer.playVideo();
```

Comment:
I have tested the cases, but it does not happen. Preload does not change the YT player state, and for complete we get ENDED state. The only time we get unstarted and cued event is when we tried to play the video.